src/openms/include/OpenMS/KERNEL/BaseFeature.h

```diff
@@ -36,11 +36,13 @@
 
 #include <OpenMS/KERNEL/RichPeak2D.h>
 #include <OpenMS/METADATA/PeptideIdentification.h>
+#include <OpenMS/METADATA/ID/IdentificationData.h>
+
+#include <boost/optional.hpp>
```

Comment:
can you do a quick check if we can just use std::optional? this seems to predate the sqitch to C17

Reply:
Yep, replaced and works.